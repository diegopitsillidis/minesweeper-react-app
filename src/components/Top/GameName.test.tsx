import React from 'react';
import { render } from "@testing-library/react";
import { GameName } from './GameName';

it('GameName renders correctly', () => {
    const { asFragment } = render(<GameName>Minesweeper</GameName>);

    expect(asFragment()).toMatchSnapshot();
});