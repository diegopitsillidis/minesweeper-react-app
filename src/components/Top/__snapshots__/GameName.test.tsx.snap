// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameName renders correctly 1`] = `
<DocumentFragment>
  <h1
    class="css-aqrpco"
  >
    Minesweeper
  </h1>
</DocumentFragment>
`;
